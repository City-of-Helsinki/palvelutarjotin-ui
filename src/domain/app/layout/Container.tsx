import classnames from 'classnames';
import React, { FunctionComponent } from 'react';

import styles from './container.module.scss';

interface Props {
  className?: string;
  size?: 'default' | 'narrow';
}

const Container: FunctionComponent<Props> = ({
  children,
  className,
  size = 'default',
}) => {
  return (
    <div className={classnames(styles.container, styles[size], className)}>
      {children}
    </div>
  );
};

export default Container;
