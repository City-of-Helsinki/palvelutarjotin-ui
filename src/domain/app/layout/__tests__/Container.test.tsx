import { render } from '@testing-library/react';
import React from 'react';

import Container from '../Container';

it('Container matches snapshot', () => {
  const { container } = render(<Container>Container children</Container>);

  expect(container.firstChild).toMatchSnapshot();
});
