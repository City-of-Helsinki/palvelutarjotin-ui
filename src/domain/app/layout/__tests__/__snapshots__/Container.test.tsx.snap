// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container matches snapshot 1`] = `
<div
  class="container default"
>
  <div>
    Container children
  </div>
</div>
`;
